kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -232,6 +232,13 @@
       "metadata": {
         "description": "The base for Kubernetes images"
       }
+    },
+    "enableTillerAddOn": {
```

Comment:
Sorry if I missed the purpose of adding tiller addon in marketplace. Let me know why we are adding this add-on

Reply:
This is to support mixed OS node case. As when Linux and windows exist then tiller install is failing (both on Azure and Azure Stack). But if you enable then things works.

Reply:
Thx

Reply:
This will truly will work when DNS issue for Azure CNI gets fixed in Azure Stack. As today the tiller is failing in nslookup on Azure Stack with Azure CNI plugin.